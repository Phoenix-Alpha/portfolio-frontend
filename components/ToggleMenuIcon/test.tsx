import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'

import { ToggleMenuIcon } from '.'

describe('<ToggleMenuIcon />', () => {
  it('should renders correct <ToggleMenuIcon />', () => {
    const wrapper = shallow(<ToggleMenuIcon />)

    expect(toJSON(wrapper)).toMatchSnapshot()
  })
})
