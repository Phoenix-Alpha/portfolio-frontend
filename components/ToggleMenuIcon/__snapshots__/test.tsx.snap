// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ToggleMenuIcon /> should renders correct <ToggleMenuIcon /> 1`] = `
<svg
  className=""
  fill="none"
  height={16}
  viewBox="0 0 22 16"
  width={22}
  xmlns="http://www.w3.org/2000/svg"
>
  <path
    d="M21 1H1M21 8H1M21 15H1"
    stroke="#ffffff"
    strokeLinecap="round"
    strokeLinejoin="round"
    strokeWidth="2"
  />
</svg>
`;
